app/pods/publications/publication/documents/route.js

```diff
@@ -15,8 +15,7 @@ export default class PublicationDocumentsRoute extends Route {
 
   async model(params) {
     // caching for use in QueryParams.queryParamsToFilter
-    // and use them in DocumentsFilterComponent
-    this.documentTypes = await this._loadDocumentTypes();
+    await this._loadDocumentTypes();
```

Comment:
Zou het niet beter zijn de load*Type-calls, in de init-hook van de route te plaatsen. Dan gebeurt de loading slechts 1 keer. Ik denk dat dit de laad-snelheid ten goede zou komen.